bedrock/mozorg/tests/test_views.py

```diff
@@ -48,20 +48,20 @@ def test_gets_default_mobilizer_url(self, resp_mock):
         ctx = resp_mock.call_args[0][2]
         self.assertEqual(ctx['mobilizer_link'], 'His Dudeness')
 
-    # testing en-US home page
+    # testing home page
     def test_en_us(self, render_mock):
         req = RequestFactory().get('/')
         req.locale = 'en-US'
         views.home(req)
         render_mock.assert_called_once_with(req,
-            'mozorg/home/home-en-US.html', ANY)
+            'mozorg/home/home.html', ANY)
 
     def test_non_en_us(self, render_mock):
```

Comment:
Instead of testing non-en-US here (which could fail as soon as `es-ES` is translated?) you can mock `lang_file_is_active` by pasting `@patch.object(views, 'lang_file_is_active', lambda *x: False)` above `def test_non_en_us`. You could rename the test to something like `test_home_old_template` too.